enterprise/server/backends/userdb/userdb_test.go

```diff
@@ -118,14 +172,8 @@ func TestCreateUser_Cloud_JoinsOnlyDomainGroup(t *testing.T) {
 	udb := env.GetUserDB()
 	ctx := context.Background()
 
-	slug := "org1-url-identifier"
-	orgGroupID, err := udb.InsertOrUpdateGroup(ctx, &tables.Group{
-		URLIdentifier: &slug,
-		OwnedDomain:   "org1.io",
-	})
-	require.NoError(t, err)
-
-	err = udb.InsertUser(ctx, &tables.User{
+	// Create the user US1; this should also create GR1.
+	err := udb.InsertUser(ctx, &tables.User{
```

Comment:
Can we use your `createUser` function here?

Reply:
Yep, also replaced the other `InsertUser` calls throughout this file.